ssl/ssl_locl.h

```diff
@@ -674,14 +674,14 @@ typedef enum tlsext_index_en {
     TLSEXT_IDX_ec_point_formats,
     TLSEXT_IDX_supported_groups,
     TLSEXT_IDX_session_ticket,
-    TLSEXT_IDX_signature_algorithms,
     TLSEXT_IDX_status_request,
     TLSEXT_IDX_next_proto_neg,
     TLSEXT_IDX_application_layer_protocol_negotiation,
     TLSEXT_IDX_use_srtp,
     TLSEXT_IDX_encrypt_then_mac,
     TLSEXT_IDX_signed_certificate_timestamp,
     TLSEXT_IDX_extended_master_secret,
+    TLSEXT_IDX_signature_algorithms,
     TLSEXT_IDX_supported_versions,
     TLSEXT_IDX_psk_kex_modes,
     TLSEXT_IDX_key_share,
```

Comment:
github won't let me comment that far away, but early_data is also empty in the clienthello.  It's 1.3-only, and you're not supposed to use it against servers that you don't know for sure support 1.3 (and if you use it the PSK extension comes last anyway and is not empty).  I guess that means it doesn't need to move, but it seemed worth noting.